joy_song/style1.css

```diff
@@ -0,0 +1,115 @@
+/* apply a google font from fonts.google.com */
```

Comment:
You can/should remove most of the comments in here. Comments are often not needed in CSS files, and comments in general are only used for clarifying code when absolutely necessary.